custom_components/tuya_local/devices/luko_dehumidifier.yaml

```diff
@@ -0,0 +1,66 @@
+name: Luko Dehumidifier
+products:
+  - id: 4w2fzgr62c1oyjsx
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+    - id: 8
+      name: current_humidity
+      type: integer
+      unit: "%"
+      class: measurement
+      readonly: true
+      range:
+        min: 15
+        max: 100
+    - id: 2
+      name: mode
+      type: string
+      mapping:
+        - dps_val: "dehumidity"
+          value: normal
+    - id: 46
+      name: humidity
+      type: integer
+      unit: "%"
+      range:
+        min: 40
+        max: 80
+      mapping:
+        - step: 5
+secondary_entities:
+  - entity: fan
+    name: Fan
+    dps:
+      - id: 1
+        name: switch
+        type: boolean
+      - id: 12
+        name: speed
+        type: string
+        mapping:
+          - dps_val: "1speed"
+            value: 20
+          - dps_val: "2speed"
+            value: 40
+          - dps_val: "3speed"
+            value: 60
+          - dps_val: "4speed"
+            value: 80
+          - dps_val: "5speed"
+            value: 100
+  - entity: sensor
+    name: Temperature
```

Comment:
change this to `class: temperature`

Reply:
updated; suggest README.md could be updated to reflect that class: temperature is an option (as well as other options).  I interpreted it as only _type_ of entity (measurement/increasing).

Reply:
class at the entity level is different than class at the dp level.  At the entity level it specifies device class (valid classes depend on the entity type, and can be found in HA documentation).  At dp level it specifies state class, which can be measurement, total, total_increasing.